common/src/warp_runner/manager/commands/constellation_commands.rs

```diff
@@ -395,6 +401,29 @@ async fn upload_files(
         };
         let local_path = Path::new(&file_path).to_string_lossy().to_string();
 
+        let file_size = match fs::metadata(&local_path) {
+            Ok(size) => size.len() as usize,
+            Err(e) => {
+                log::error!("Not possible to get file size, error: {}", e);
+                continue;
+            }
+        };
+
+        if (current_size_ipfs + file_size) > max_size_ipfs {
```

Comment:
Do note that internally it does check for this and would return an error ahead of time before any upload attempts.